tests/test_peak_merging.py

```diff
@@ -48,3 +48,29 @@ def test_replace_merged(intervals, merge_instructions):
         assert x in result, "Didn't put in merged"
     for x in result:
         assert np.isin(x, merged_itvs) or np.isin(x, kept_itvs), "Invented itv"
+
+
+@hypothesis.given(fake_hits,
+                  hypothesis.strategies.integers(min_value=0, max_value=100))
+@hypothesis.settings(deadline=None)
+def test_add_lone_hits(hits, peak_length):
+    peak = np.zeros(1, dtype=strax.peak_dtype())
+    peak['length'] = peak_length
+    peak['dt'] = 1
+
+    to_pe = np.ones(10000)
```

Comment:
```suggestion
    to_pe = np.ones(np.max(hits['channel']))
```

Reply:
I had this before :D That is why the previous version was failing. In case hits are empty np.max has no identity. Hence I put 10k to avoid some more complicated if else thingy.

Reply:
haha okay, I see

Reply:
are you sure by the way? I see that there was a typo in `hits` that was the issue. But okay, it's really not much of an issue

Reply:
Yes, I corrected the typo first and hit then the other issue.